conf/start/zentral/base.json

```diff
@@ -89,6 +89,8 @@
     "zentral.contrib.osquery": {},
     "zentral.contrib.santa": {
       "metrics": true
-    }
+    },
```

Comment:
Not sure about activating this app by default.

Reply:
You should not be using the default config, but the devmdm one, or another custom one.